chapter-06/inprogress/bookworm_internal_test.go

```diff
@@ -66,13 +66,12 @@ func TestFindCommonBooks(t *testing.T) {
 		input []Bookworm
 		want  []Book
 	}{
-		// TODO FIX ME
 		"no common book": {
 			input: []Bookworm{
 				{Name: "Fadi", Books: []Book{handmaidsTale, theBellJar}},
 				{Name: "Peggy", Books: []Book{oryxAndCrake, janeEyre}},
 			},
-			want: []Book{},
+			want: nil,
```

Comment:
Do we explain what the differences are?

Reply:
I think we talk about the empty value...